airflow/contrib/operators/gcs_download_operator.py

```diff
@@ -44,11 +49,23 @@ def __init__(
         self.bucket = bucket
         self.object = object
         self.filename = filename
+        self.store_to_xcom = store_to_xcom
         self.google_cloud_storage_conn_id = google_cloud_storage_conn_id
         self.delegate_to = delegate_to
 
     def execute(self, context):
-        logging.info('Executing download: %s, %s, %s', self.bucket, self.object, self.filename)
-        hook = GoogleCloudStorageHook(google_cloud_storage_conn_id=self.google_cloud_storage_conn_id,
-                                      delegate_to=self.delegate_to)
-        print(hook.download(self.bucket, self.object, self.filename))
+        if self.store_to_xcom:
```

Comment:
We should also enforce a file-size cap on the file_bytes if xcom is enabled. We don't want to put large files in the DB. I checked the xcom DB, and it's using BLOB as its type, which (at least in MySQL) has a max size of 65535. I think we should probably enforce 48KB as the max to allow for whatever pickling overhead there is.